src/library/scala/collection/SortedOps.scala

```diff
@@ -12,6 +12,10 @@ trait SortedOps[A, +C] {
   /** Returns the last key of the collection. */
   def lastKey: A
 
+  /** Comparison function that orders keys. */
+  @deprecated("Use Ordering.compare instead.", "2.13.0")
+  @inline def compare(k0: A, k1: A): Int = ordering.compare(k0, k1)
```

Comment:
We should also mark it `@deprecatedOverriding`.

Reply:
updated. thanks